repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/publicationlandingpage-main.ftl

```diff
@@ -18,182 +18,170 @@
+            <h3>
+                <a class="nhsuk-resources__link" href="${fileURL}" title="${docLink.filename}">
+                    ${docLink.filename?keep_before_last(".")} - ${getDocumentFormat(docLink.filename?keep_after_last("."))}
+                </a>
+            </h3>
+
+            <@docDetailBlock
+                publishedDate=docLink.properties['jcr:created']
+                updatedDate=docLink.lastModified
+                fileType=docLink.filename?keep_after_last(".")
+                fileLengthInKB=docLink.lengthKB />
+        </div>
+    </div>
 </#macro>
 
 <#--  Renders document detail block  -->
 <#macro docDetailBlock publishedDate updatedDate fileType fileLengthInKB=0>
-    <div class="nhsuk-review-date" style="margin-top:0px">
-        <p class="nhsuk-body-s">
-            Published: ${getDefaultFormattedDate(publishedDate)}<br>
-            <#if updatedDate?has_content>Updated: ${getDefaultFormattedDate(updatedDate)}<br></#if>
-            ${fileType?upper_case}${(fileType = 'WEB')?then('',', ' + fileLengthInKB + 'kB')}
-        </p>
+    <span>Published: ${getDefaultFormattedDate(publishedDate)}</span>
+    <#if updatedDate?has_content><span>Updated: ${getDefaultFormattedDate(updatedDate)}</span></#if>
+    <span>${fileType?upper_case}${(fileType = 'WEB')?then('',', ' + fileLengthInKB + 'kB')}</span>
+
+</#macro>
+
+<#macro docIcon fileType>
+    <div class="hee-publication-doc__wrapper">
+        <div class="hee-publication-doc__icon">
+            <div class="hee-publication-doc__icon__page"></div>
+            <div class="hee-publication-doc__icon__corner"></div>
+        </div>
+        <div class="hee-publication-doc__icon__title">${fileType}</div>
+
     </div>
 </#macro>
 
 <#if document??>
-    <div class="nhsuk-width-container">
-        <main id="maincontent" role="main" class="nhsuk-main-wrapper">
+    <main id="maincontent" role="main" class="page page--rightbar">
+        <#--  Renders title & subtitle  -->
+        <div class="page__header">
             <div class="nhsuk-width-container">
-
-                <#--  Renders title & subtitle  -->
-                <div class="nhsuk-grid-row">
-                    <div class="nhsuk-grid-column-two-thirds">
-                        <h1>
-                            <span role="text">${document.title}
-                                <span class="nhsuk-caption-xl nhsuk-caption--bottom">
-                                    ${document.subtitle!}
-                                </span>
-                            </span>
-                        </h1>
-                    </div>
+                <h1>${document.title}</h1>
+                <span class="nhsuk-caption-xl">${document.subtitle!}</span>
+            </div>
+        </div>
+
+        <div class="nhsuk-width-container">
+            <div class="page__layout">
+                <div class="page__main">
+                    <#-- Author and published date -->
```

Comment:
Could we remove `Author and published date` block as I couldn't see it either in figma or in the FE (unless it is agreed separately as part of a ticket)?